src/generated/resources/schema.graphql

```diff
@@ -104,6 +137,8 @@ type Query {
   directConnections: [DirectConnection]!
   "Find CCloud Kafka clusters using a connection and various criteria"
   findCCloudKafkaClusters(connectionId: String!, environmentId: String = "", name: String = "", provider: String = "", region: String = ""): [CCloudKafkaCluster]!
+  "Get Flink compute pools for a specific connection and environment"
+  getFlinkComputePools(connectionId: String!, envId: String!): [FlinkComputePool]!
```

Comment:
From the extension perspective, we don't use `findCCloudKafkaClusters` and lean more on `ccloudConnectionById` to get the environments and sub-resources. Could the compute pools be exposed through `CCloudEnvironment` instead of having a separate query? https://github.com/confluentinc/ide-sidecar/blob/7161e6bb59afbf8fcb1c374126f7c243853d2cb3/src/generated/resources/schema.graphql#L9-L17